docs/guide/core-concepts.rst

```diff
@@ -13,8 +13,8 @@ alike. If GIS is brand, brand new to you, `this
 document <https://www.gislounge.com/what-is-gis/>`__ is a useful high
 level overview.
 
-Basic Terms
-===========
+Glossary
+========
 
 -  **Tile:** A grid of numeric *cells* that represent some data on the
    Earth.
```

Comment:
I can't comment on the exact line since it wasn't changed, but I think that `Layer` should be `Tile Layer` since that's how it's referred to throughout the document. With a mention that a `Tile Layer` can also be called a `Layer`